src/bind.rs

```diff
@@ -35,20 +36,29 @@ impl BindMount {
             source: path_to_cstring(source.as_ref()),
             target: path_to_cstring(target.as_ref()),
             recursive: true,
+            readonly: false,
         }
     }
     /// Toggle recursion
     pub fn recursive(mut self, flag: bool) -> BindMount {
         self.recursive = flag;
         self
     }
+    /// Set readonly flag
+    pub fn readonly(mut self, flag: bool) -> BindMount {
+        self.readonly = flag;
+        self
+    }
 
     /// Execute a bind mount
     pub fn bare_mount(self) -> Result<(), OSError> {
         let mut flags = flags::MS_BIND;
         if self.recursive {
             flags = flags | flags::MS_REC;
         }
+        if self.readonly {
+            flags = flags | flags::MS_RDONLY;
```

Comment:
This is useless. It doesn't work. You have to remount later to make a readonly bind mount.